# Generated by Django 4.2 on 2025-03-02 16:21

from django.db import migrations, models
import django.utils.timezone


class Migration(migrations.Migration):

    dependencies = [
        ('error_logging', '0003_alter_error_error_time'),
    ]

    operations = [
        migrations.CreateModel(
            name='StackTrace',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('error_id', models.CharField(max_length=255)),
                ('stack_trace', models.TextField()),
                ('timestamp', models.DateTimeField(default=django.utils.timezone.now)),
                ('additional_data', models.JSONField(blank=True, null=True)),
                ('trace_level', models.CharField(choices=[('DEBUG', 'Debug'), ('INFO', 'Info'), ('WARNING', 'Warning'), ('ERROR', 'Error'), ('CRITICAL', 'Critical')], default='ERROR', max_length=10)),
            ],
            options={
                'db_table': 'Error_Stacktrace',
            },
        ),
    ]
